src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIService.kt

```diff
@@ -48,6 +57,43 @@ class CommunityAPIService(
   }
 }
 
+@Service
+class CommunityAPIAppointmentEventService(
+  @Value("\${interventions-ui.baseurl}") private val interventionsUIBaseURL: String,
+  @Value("\${interventions-ui.locations.session-feedback}") private val interventionsUISessionFeedbackLocation: String,
+  @Value("\${community-api.locations.patch-appointment}") private val communityAPIPatchAppointmentLocation: String,
+  @Value("\${community-api.integration-context}") private val integrationContext: String,
+  private val communityAPIClient: CommunityAPIClient,
+) : ApplicationListener<AppointmentEvent>, CommunityAPIService {
+  companion object : KLogging()
+
+  override fun onApplicationEvent(event: AppointmentEvent) {
+    when (event.type) {
+      SESSION_FEEDBACK_RECORDED -> {
+        val url = UriComponentsBuilder.fromHttpUrl(interventionsUIBaseURL)
+          .path(interventionsUISessionFeedbackLocation)
+          .buildAndExpand(event.appointment.actionPlan.referral.id, event.appointment.sessionNumber)
+          .toString()
+
+        val jsonPatch = JsonPatch(
+          listOf(
+            ReplaceOperation(of("notes"), valueOf(url)),
```

Comment:
this is neat - is this a new thing you guys have implemented in community api?

Reply:
the json patch format that is

Reply:
This is how the Delius-api is written to accept jsonPatch and so it was thought it would also work well for community-api too. However, Prisons teams are suggesting PUT/POST is much more informative.